BedrockServer.cpp

```diff
@@ -369,11 +369,13 @@ void BedrockServer::sync(const SData& args,
             // Store this before we start writing to the DB, which can take a while depending on what changes were made
             // (for instance, adding an index).
             upgradeInProgress.store(true);
+            command.request.methodLine = "_upgradeDB";
             if (server._upgradeDB(db)) {
                 server._syncThreadCommitMutex.lock();
                 committingCommand = true;
                 server._syncNode->startCommit(SQLiteNode::QUORUM);
                 server._lastQuorumCommandTime = STimeNow();
+                SDEBUG("finished sending distributed transaction for db upgrade");
```

Comment:
Capitalize `Finished` and add a `.` to the end. This could be done to a lot of the log lines you added, none of them are formatted like actual sentences.